ui/imports/shared/popups/send/SendModal.qml

```diff
@@ -316,25 +299,69 @@ StatusDialog {
-                        collectiblesModel: popup.preSelectedAccount ? popup.nestedCollectiblesModel : null
-                        networksModel: popup.store.flatNetworksModel
-                        visible: (!!d.selectedHolding && d.selectedHoldingType !== Constants.TokenType.Unknown) ||
-                                 (!!d.hoveredHolding && d.hoveredHoldingType !== Constants.TokenType.Unknown)
-                        onItemSelected: {
-                            d.setSelectedHoldingId(holdingId, holdingType)
+
+                        property var selectedItem
+                        property bool onlyAssets: false
+
+                        assetsModel: assetsAdaptor.outputAssetsModel
+                        collectiblesModel: collectiblesAdaptorLoader.active
+                                           ? collectiblesAdaptorLoader.item.model : null
+
+                        TokenSelectorViewAdaptor {
+                            id: assetsAdaptor
+
+                            assetsModel: popup.store.walletAssetStore.groupedAccountAssetsModel
+
+                            flatNetworksModel: popup.store.flatNetworksModel
+                            currentCurrency: popup.store.currencyStore.currentCurrency
+                            accountAddress: popup.preSelectedAccount ? popup.preSelectedAccount.address : ""
+                            showCommunityAssets: popup.store.tokensStore.showCommunityAssetsInSend
                         }
-                        onSearchTextChanged: assetsAdaptor.assetSearchString = assetSearchString
-                        formatCurrentCurrencyAmount: function(balance){
-                            return popup.store.currencyStore.formatCurrencyAmount(balance, popup.store.currencyStore.currentCurrency)
+
+                        Loader {
+                            id: collectiblesAdaptorLoader
+
+                            active: !d.isBridgeTx
+
+                            sourceComponent: CollectiblesSelectionAdaptor {
+                                accountKey: popup.preSelectedAccount ? popup.preSelectedAccount.address : ""
+                                collectiblesModel: collectiblesStore
+                                                   ? collectiblesStore.jointCollectiblesBySymbolModel
+                                                   : null
+                            }
                         }
-                        formatCurrencyAmountFromBigInt: function(balance, symbol, decimals){
-                            return popup.store.formatCurrencyAmountFromBigInt(balance, symbol, decimals, {noSymbol: true})
+
+                        onAssetSelected: {
+                            const entry = ModelUtils.getByKey(
+                                            assetsModel, "tokensKey", key)
+                            d.selectedHoldingType = Constants.TokenType.ERC20
+                            d.selectedHolding = entry
+                            selectedItem = entry
```

Comment:
As mentioned above. It can be simplified but right now too much depends on that. This change will be much easier after refactoring subcomponents which complicate it a lot.